.github/workflows/drivers.yml

```diff
@@ -58,7 +49,7 @@ jobs:
           CHECKOUT_BEFORE_PATCHING: false
 
         run: |
-          git fetch
+          git fetch -a
```

Comment:
Are we overkilling here? Why the `-a` if we've already checked out with `fetch-depth: 0`?

Reply:
ah yeah this was me being a bit overzealous when the `git show` command wasn't working... I'll fix it!